pkg/commands/test/test.go

```diff
@@ -79,6 +79,7 @@ func NewTestCommand(osExit func(int), getOutputManager func() OutputManager) *co
 				osExit(1)
 			}
 
+			foundFailures := false
 			var res CheckResult
 			if viper.GetBool(CombineConfigFlagName) {
 				res, err = processData(ctx, configurations, compiler)
```

Comment:
The lines dealing with combined-config + single file evaluation are quite similar. The only difference is in the `out.Put` call and what is passed to the `processData` command. That could be combined in a single function.

Reply:
Thanks, good suggestion. I've made this change and pushed.